docs/developer-notes.md

```diff
@@ -76,6 +76,53 @@ Scuri uses [Jasmine's spy feature](https://jasmine.github.io/2.0/introduction.ht
 We loosely follow the [Jasmine Style Guide](https://github.com/CareMessagePlatform/jasmine-styleguide) when writing tests. 
 One main guideline is that a `describe` block should be created for each method / scenario under test, and an `it` block should be created for each property being verified.
 
+# E2E Testing
+
+## Running E2E Tests
+
+E2E Tests can be executed using `npm run e2e`. You should see CATcher launch in an instance of Google Chrome, with some automated actions occurring on it. Note: Google Chrome needs to be installed on the machine.
+
+Unlike the production version of CATcher, we do not use the actual GitHub API in the E2E tests. Mock data is used to simulate the GitHub API. You can run `npm run ng:serve:test` to run CATcher in this "offline" mode (to further develop or debug the E2E tests).
+The following additional parameters would allow for further customisation,
+
+| Additional Parameter | Default | Description | Full Command Example | Command Explanation
+| :---: | :---: | :-----: | :-------: | :------: |
+| `--protractor-config=e2e/protractor.*.conf.js` | `potractor.conf.js` | Allows selection of the Protractor configuration file | `npm run e2e -- --protractor-config=e2e/protractor.firefox.conf.js` | Runs E2E Tests on the Firefox Browser |
+| `--suite=*` | All Suites | Runs E2E Tests for specific suites | `npm run e2e -- --suite=login,bugReporting` | Run E2E Tests from Login and BugReporting Suites only |
+
+**TO NOTE:**
+- Relevant Browsers must be installed prior to running tests (i.e. Chrome, Firefox).
+- CATcher can be launched in the Test Build by using the command `npm run ng:serve:test` to further develop mock services and debug E2E Tests.
```

Comment:
Let's remove this sentence as this information is already mentioned above.

Reply:
Ah yes sure 👍